templates/razorpay-settings-templates.php

```diff
@@ -156,6 +156,48 @@ function displayPaymentAction()
         echo $paymentAction;
     }
 
+    /**
+     * Currency field of settings page
+    **/
+    function displayCurrencyAction()
+    {
+        $default = get_option('currency_action_field');
+
+        $supported_currencies = json_decode(file_get_contents(__DIR__ . "/../supported-currencies.json"), true)['supported-currencies'];
+
+        $supported_currency_codes = [];
+
+        foreach ($supported_currencies as $k => $v)
+        {
+            array_push($supported_currency_codes, $v['iso_code']);
+        }
+
+        $currency = '<select name="currency_action_field" id="currency_action"> ';
+                    
+        foreach ($supported_currency_codes as $index => $value)
+        { 
+            if ($value === $default)
+            {
+                $currency .= '<option value="' . $value . '" selected="selected">' . $value . '</option>'; 
+            }                        
+            else
+            {
+                if (empty($default) === true && $value === 'INR')
```

Comment:
use and instead &&

Reply:
fixed here https://github.com/razorpay/razorpay-quick-payments/blob/4bf92b0b3e7bec74ab0bb3ee9f469fb902639578/templates/razorpay-settings-templates.php#L185